docs/tutorial.rst

```diff
@@ -3,6 +3,113 @@
+
+***************************
+Using the ``stdpopsim`` CLI
+***************************
+In order to use the ``stdpopsim`` CLI the ``stdpopsim`` package must be
+installed (see :ref:`Installation <sec_installation>`). The CLI provides access
+to the :ref:`catalog <sec_catalog>` of models that have already been implemented
+by ``stdpopsim``. The first step for using the CLI is to select the species that
+you are interested in simulating data for. In order to see which species are
+available run
+
+.. code-block:: console
+
+    $ stdpopsim --help
+
+This shows the species currently supported by ``stdpopsim``. This means that
+``stdpopsim`` knows various traits of these species including chromosome size
+and recombination rates. Once we've selected a species, in this case humans, we
+can look at the help again as follows.
+
+.. code-block:: console
+
+    $ stdpopsim homsap --help
+
+For conciseness we do not show the output here but this time you should see a
+different output which shows options for performing the simulation itself and
+the species default parameters. This includes selecting the demographic model,
+chromosome, recombination map, and number of samples. 
+
+The most basic simulation we can run is to draw two samples using the species'
+defaults as seen in the species help (``stdpopsim homsap --help``). These
+defaults include constant size population, uniform recombination map, and a
+default chromosome. To save time we will also specify that the simulation use
+chromosome 22 with the ``-c`` option. 
+
+.. code-block:: console
+
+    $ stdpopsim homsap -c chr22 -o foo.ts 2
+
+.. warning:: It's important to remember to either redirect the output of ``stdpopsim``
+                to file or to use the ``-o/--output`` option. If you do not, the 
+                binary output may mess up your terminal session.
+
+Say we want to use a specific demographic model. We look up the available models
+using the ``--help-models`` flag:
+
+.. code-block:: console
+
+    $ stdpopsim homsap --help-models
+
+This gives all of the possible demographic models we could simulate. We choose
+the the two population out-of-Africa model from `Tennesen et al.
+<https://doi.org/10.1126/ science.1219240>`_ . By looking at the model help we
+find that the name for this model is ``ooa_2`` and that we can specify it using
+the ``--model`` option. We choose to draw two samples from the African
+population and three samples from the European population. To increase
+simulation speed we can also chose to simulate only a subset of the chromosome
+using the ``-l`` option (e.g. 5%). The command now looks like this:
+
+.. code-block:: console
+
+    $ stdpopsim homsap -c chr22 -l 0.05 -o foo.ts --model ooa_2 2 3
+
+Note that there are now two numbers after the model option. This is because the
+model simulates two populations so we have to specify a number of samples to
+draw from each population (the order of those numbers is the same as the order
```

Comment:
I'd be even more explicit and say what this means for your example simulation: 'here, for instance, we simulate 2 samples from population X and 3 samples from population Y'.
See also the conversation in #151.